.circleci/config.yml

```diff
@@ -50,9 +50,20 @@ jobs:
       - run:
           name: Check for changes
           command: git diff --exit-code
+  check-security-configs:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    steps:
+      - checkout
+      - run:
+          name: check security configs
+          command: |
+            forge install --no-git foundry-rs/forge-std
+            forge script CheckSecuityConfigs --rpc-url=https://ethereum.publicnode.com
```

Comment:
We have some nodes that we can use as part of CI here: https://github.com/ethereum-optimism/superchain-ops/blob/3ba164e1c5e5781ba95d588b57f393159188018f/.circleci/config.yml#L3